.gitignore

```diff
@@ -1,6 +1,7 @@
 *.gem
 *.rbc
 *.sqlite3
+Gemfile.lock
```

Comment:
Remove this line
http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/